weaver/cli.py

```diff
@@ -1303,19 +1389,175 @@ def execute(self,
+            The subscriber keys indicate which type of subscriber and for which status the notification will be sent.
+            Note that modifying this value will override any previously configured subscribers on the job.
+            Therefore, any update must re-submit any previously submitted subscribers to achieve an 'additive' result.
+            If an empty mapping is submitted, all subscribers will be cleared.
+        :param url: Instance URL if not already provided during client creation.
+        :param auth:
+            Instance authentication handler if not already created during client creation.
+            Should perform required adjustments to request to allow access control of protected contents.
+        :param headers:
+            Additional headers to employ when sending request.
+            Note that this can break functionalities if expected headers are overridden. Use with care.
+        :param with_links: Indicate if ``links`` section should be preserved in returned result body.
+        :param with_headers: Indicate if response headers should be returned in result output.
+        :param request_timeout: Maximum timout duration (seconds) to wait for a response when performing HTTP requests.
+        :param request_retries: Amount of attempt to retry HTTP requests in case of failure.
+        :param output_format: Select an alternate output representation of the result body contents.
+        :param output_refs:
+            Indicates which outputs by ID to be returned as HTTP Link header reference instead of body content value.
+            With reference transmission mode, outputs that contain literal data will be linked by ``text/plain`` file
+            containing the data. outputs that refer to a file reference will simply contain that URL reference as link.
+            With value transmission mode (default behavior when outputs are not specified in this list), outputs are
+            returned as direct values (literal or href) within the response content body.
+        :param output_filter:
+            Indicates a list of outputs to omit from the results. If unspecified (default), all outputs are returned.
+        :param output_context:
+            Specify an output context for which the `Weaver` instance should attempt storing the :term:`Job` results
+            under the nested location of its configured :term:`WPS` outputs. Note that the instance is not required
+            to fulfill that preference, and can ignore this value if it deems that the provided context is inadequate.
+        :return: Result of the :term:`Job` update operation.
+        """
+        job_id, job_url = self._parse_job_ref(job_reference, url)
+        LOGGER.info("Attempting job trigger: [%s]", job_id)
+        job_res_url = f"{job_url}/results"
+        update_data = {}
+        update_headers = {}
+        update_headers.update(self._headers)
+        if title or title is None:
+            update_data["title"] = title
```

Comment:
It is needed in this case because updating with an explicit `None` unsets the title, but we don't want to inject the "unspecified" default of `null` that would create an invalid JSON content.